backend/src/components/document.js

```diff
@@ -0,0 +1,73 @@
+'use strict';
+const { MappableObjectForFront, MappableObjectForBack } = require('../util/mapping/MappableObject');
+const { ApplicationDocumentsMappings, DocumentsMappings } = require('../util/mapping/Mappings');
+const { postApplicationDocument, getApplicationDocument, deleteDocument, patchOperationWithObjectId } = require('./utils');
+const HttpStatus = require('http-status-codes');
+const log = require('./logger');
+
+const { getFileExtension, convertHeicDocumentToJpg } = require('../util/uploadFileUtils');
+
+async function createApplicationDocuments(req, res) {
+  try {
+    const documents = req.body;
+    for (let document of documents) {
+      let payload = new MappableObjectForBack(document, ApplicationDocumentsMappings).toJSON();
+      payload.ccof_applicationid = document.ccof_applicationid;
+      payload.ccof_facility = document.ccof_facility;
```

Comment:
/api/ApplicationDocument is a custom D365 endpoint, and for some reason, the facilityId field has different names in the GET and POST requests. That's why I manually set it for the POST request instead of using Mapping.